resources/zib/zib-Vaccination-request.xml

```diff
@@ -0,0 +1,95 @@
+  <purpose value="This ImmunizationRecommendation resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Vaccination-v4.0(2020EN)](https://zibs.nl/wiki/Vaccination-v4.0(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-vaccination-v4.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/Vaccination-v4.0(2020EN)" />
+    <name value="zib Vaccination-v4.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="ImmunizationRecommendation" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/ImmunizationRecommendation" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="ImmunizationRecommendation">
+      <path value="ImmunizationRecommendation" />
+      <short value="Vaccination" />
+      <comment value="The zib Vaccination is mapped to this ImmunizationRecommendation profile and a profile on Immunization (http://nictiz.nl/fhir/StructureDefinition/zib-Vaccination-event) to align with the intention of FHIR. All past vaccinations are covered using the Immunization profile, while all future vaccinations, including the advised vaccinations, are covered in this ImmunizationRecommendation profile. Both profiles contain the zib mappings, except for the Dose (NL-CM:11.1.4) and Administrator (NL-CM:11.1.6) concepts which are, based on their current definition, not suitable for future vaccinations. Therefore they have only been mapped to the Immunization profile.&#xA;&#xA;Please note that the ImmunizationRecommendation profile has been aligned as much as possible with the zib Vaccination by setting the cardinalities of `.recommendation` ,`.vaccineCode` and `.dateCriterion` to x..1. This results in the removal of the ability to exchange a vaccination schema. Moreover, ImmunizationRecommendation has more mandatory elements that are not represented in the zib, such as `.date` and `.recomendation.forecastStatus`. Implementers will need to take these requirements into account when implementing the future vaccinations.&#xD;&#xA;&#xD;&#xA;The zib centre has acknowledged that the Vaccination zib is currently ill-defined, especially concerning future vaccinations. Therefore the zib will be completely redesigned for a future release." />

```

Comment:
De feedback op -event hier toegepast:

> The zib Vaccination is mapped to this ImmunizationRecommendation profile and a profile on Immunization (http://nictiz.nl/fhir/StructureDefinition/zib-Vaccination-event). All administered vaccinations are covered using this Immunization profile, while all planned vaccinations are covered in the ImmunizationRecommendation profile. This means that zib concepts can exist in both profiles. The Dose (NL-CM:11.1.4) and Administrator (NL-CM:11.1.6) concepts are not suitable for planned vaccinations based on their current definitions and are therefore not mapped in this ImmunizationRecommendation profile.

> Please note that the ImmunizationRecommendation profile has been aligned as much as possible with the zib Vaccination by setting the maximum cardinalities of `.recommendation` ,`.vaccineCode` and `.dateCriterion` to 1. This results in the removal of the ability to exchange a vaccination schema. Moreover, ImmunizationRecommendation has mandatory elements that are not represented in the zib, such as `.date` and `.recomendation.forecastStatus`. Implementers will need to take these requirements into account when implementing the future vaccinations.

(Kleine nuanceverbeteringen in 2e paragraaf, 3e paragraaf bewust weggelaten - is het nodig om in dit profiel in te gaan op eventuele potentiële mogelijke wijzigingen in de zib in de toekomst?